data/campaigns/Eastern_Invasion/scenarios/11_Captured.cfg

```diff
@@ -1084,12 +1084,13 @@
         [objectives]
             side=1
             [objective]
-                description= _ "Gweddry reaches the western exit"
+                description= _ "Move Gweddry to the western exit"
                 condition=win
             [/objective]
             [objective]
                 description= _ "Retrieve the stolen gold"
                 condition=win
+                {BONUS_OBJECTIVE_CAPTION}
```

Comment:
Not sure about labelling this a bonus - the next level is a bloodbath even with the extra gold. Is the next level even possible without the extra gold, assuming that the player's skill level means that they leave the gold behind on this level?